bat/lib/mixerlib.bash

```diff
@@ -86,19 +86,58 @@ mixer-bundle-remove() {
   mixer $MIXARGS bundle remove $1 --native=true
 }
 
-download-rpm() {
-  mkdir -p $BATS_TEST_DIRNAME/local-rpms
-  pushd $BATS_TEST_DIRNAME/local-rpms
-  curl -LO $1
-  popd
+get-current-version() {
+  latest=$(curl https://download.clearlinux.org/latest)
+
+  echo $latest
 }
 
-get-last-format-boundary() {
-  latest=$(curl https://download.clearlinux.org/latest)
+get-current-format() {
+  latest=$(get-current-version)
   format=$(curl https://download.clearlinux.org/update/$latest/format)
+
+  echo $format
+}
+
+get-last-format-boundary() {
+  format=$(get-current-format)
   first=$(curl https://download.clearlinux.org/update/version/format$format/first)
 
   echo $(($first-10))
 }
 
+setup-dnf() {
```

Comment:
this gets called for every time you `download_rpm`. Can you put this in a shared location and check for existence so you don't have to write the same file again and again? It doesn't change.

Reply:
You mean something like create a `bat/config` folder and keep it there instead of creating it all the time? It sounds better indeed.

Reply:
Yep!